eth/downloader/statesync.go

```diff
@@ -428,20 +440,34 @@ func (s *stateSync) assignTasks() {
 
 // fillTasks fills the given request object with a maximum of n state download
 // tasks to send to the remote peer.
-func (s *stateSync) fillTasks(n int, req *stateReq) []common.Hash {
+func (s *stateSync) fillTasks(n int, req *stateReq) ([]common.Hash, [][][]byte, []common.Hash) {
```

Comment:
With three un-named return values, please document them in the method doc

Reply:
Added named returns.